packages/enzyme/src/Debug.js

```diff
@@ -66,6 +94,7 @@ function indentChildren(childrenStrs, indentLength) {
 
 export function debugNode(node, indentLength = 2, options = {}) {
   if (typeof node === 'string' || typeof node === 'number') return escape(node);
+  if (typeof node === 'function') return '[function child]';
```

Comment:
why is this necessarily a function child?

Reply:
I'm not sure what else to call a `props.children` that's a function. Perhaps cases also hit here, but i'm not sure what they are.

Reply:
right but `debugNode` won't necessarily be called on a child.